django/db/backends/oracle/base.py

```diff
@@ -49,11 +51,20 @@ def _setup_environment(environ):
 
 
 try:
-    import cx_Oracle as Database
+    import oracledb as Database
 except ImportError as e:
-    raise ImproperlyConfigured("Error loading cx_Oracle module: %s" % e)
+    try:
+        import cx_Oracle as Database
+
+        warnings.warn(
+            "cx_Oracle is deprecated. Please use oracledb instead.",
+            RemovedInDjango51Warning,
```

Comment:
```suggestion
            RemovedInDjango51Warning,
            stacklevel=2,
```
This is Django's usual practice, see e.g. ticket-18127.